<!-- test-analysis-auto -->
@claude 以下のテスト失敗を修正してください。

**失敗ジョブ**: Lint
**Workflow Run**: https://github.com/sy-hash/mind-echo/actions/runs/23027717930

```
Lint	Run swift-format lint	2026-03-12T22:53:35.9435440Z MindEcho/MindEcho/ViewModels/TranscriptionViewModel.swift:47:1: error: [LineLength] line is too long
Lint	Run swift-format lint	2026-03-12T22:53:35.9637690Z MindEcho/MindEcho/ViewModels/HomeViewModel.swift:52:1: error: [LineLength] line is too long
Lint	Run swift-format lint	2026-03-12T22:53:36.0389390Z MindEcho/MindEcho/Services/SummarizationService.swift:21:1: error: [LineLength] line is too long
Lint	Run swift-format lint	2026-03-12T22:53:36.0515020Z MindEcho/MindEcho/Services/SummarizerPreference.swift:41:20: error: [AddLines] add 1 line break
Lint	Run swift-format lint	2026-03-12T22:53:36.0811000Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:54:1: error: [Indentation] indent by 1 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0811790Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:55:1: error: [Indentation] indent by 1 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0812410Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:56:1: error: [Indentation] indent by 1 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0813130Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:56:32: error: [AddLines] add 1 line break
Lint	Run swift-format lint	2026-03-12T22:53:36.0813890Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:61:38: error: [AddLines] add 1 line break
Lint	Run swift-format lint	2026-03-12T22:53:36.0814570Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:62:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0815280Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:63:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0815970Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:64:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0816660Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:65:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0817370Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:66:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0818060Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:67:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0818800Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:68:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0819470Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:69:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0820180Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:70:1: error: [Indentation] indent by 4 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0820870Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:75:1: error: [Indentation] unindent by 2 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0821580Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:76:1: error: [Indentation] unindent by 2 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0822750Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:77:1: error: [Indentation] unindent by 2 spaces
Lint	Run swift-format lint	2026-03-12T22:53:36.0823450Z MindEcho/MindEcho/Services/OpenAISummarizationService.swift:78:1: error: [Indentation] unindent by 2 spaces
```

テストが通るようにコードを修正し、ビルドが通ることも確認してください。

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -110,7 +110,7 @@ xcodebuild test \
 
 Apple 公式の [swift-format](https://github.com/swiftlang/swift-format) を使用してコードスタイルを統一しています。設定はプロジェクトルートの `.swift-format` を参照。
 
-- コミット前にフォーマットを適用すること: `xcrun swift-format format --in-place --recursive --configuration .swift-format <対象ディレクトリ>`
+- コミット前にフォーマットを適用すること: `xcrun swift-format format --in-place --recursive <対象ディレクトリ>`
 
 ### Swift Concurrency
 

--- a/MindEcho/MindEcho/Services/OpenAISummarizationService.swift
+++ b/MindEcho/MindEcho/Services/OpenAISummarizationService.swift
@@ -51,31 +51,33 @@ struct OpenAISummarizationService: Sendable {
         if httpResponse.statusCode != 200 {
             // OpenAI エラーレスポンスからユーザー向けメッセージを抽出
             if let json = try? JSONSerialization.jsonObject(with: data) as? [String: Any],
-               let errorDict = json["error"] as? [String: Any],
-               let message = errorDict["message"] as? String,
-               !message.isEmpty {
+                let errorDict = json["error"] as? [String: Any],
+                let message = errorDict["message"] as? String,
+                !message.isEmpty
+            {
                 throw SummarizationError.apiError(message)
             }
 
             // ステータスコードに応じた定型文にフォールバック
-            let userMessage: String = switch httpResponse.statusCode {
-            case 401:
-                "API キーが無効か、認証に失敗しました。設定を確認してください。"
-            case 429:
-                "リクエストが多すぎます。しばらく時間をおいてから再試行してください。"
-            case 500...599:
-                "OpenAI API 側でエラーが発生しました。時間をおいて再試行してください。"
-            default:
-                "HTTP \(httpResponse.statusCode) エラーが発生しました。"
-            }
+            let userMessage: String =
+                switch httpResponse.statusCode {
+                case 401:
+                    "API キーが無効か、認証に失敗しました。設定を確認してください。"
+                case 429:
+                    "リクエストが多すぎます。しばらく時間をおいてから再試行してください。"
+                case 500...599:
+                    "OpenAI API 側でエラーが発生しました。時間をおいて再試行してください。"
+                default:
+                    "HTTP \(httpResponse.statusCode) エラーが発生しました。"
+                }
             throw SummarizationError.apiError(userMessage)
         }
 
         guard let json = try JSONSerialization.jsonObject(with: data) as? [String: Any],
-              let choices = json["choices"] as? [[String: Any]],
-              let firstChoice = choices.first,
-              let message = firstChoice["message"] as? [String: Any],
-              let content = message["content"] as? String
+            let choices = json["choices"] as? [[String: Any]],
+            let firstChoice = choices.first,
+            let message = firstChoice["message"] as? [String: Any],
+            let content = message["content"] as? String
         else {
             throw SummarizationError.invalidResponse
         }

--- a/MindEcho/MindEcho/Services/SummarizationService.swift
+++ b/MindEcho/MindEcho/Services/SummarizationService.swift
@@ -18,7 +18,9 @@ struct SummarizationService {
 
     // MARK: - Dispatch (共通エントリポイント)
 
-    static func summarize(text: String, instruction: String, type: SummarizerType, apiKey: String) async throws -> String {
+    static func summarize(
+        text: String, instruction: String, type: SummarizerType, apiKey: String
+    ) async throws -> String {
         switch type {
         case .onDevice:
             try await SummarizationService().summarize(text: text, instruction: instruction)

--- a/MindEcho/MindEcho/Services/SummarizerPreference.swift
+++ b/MindEcho/MindEcho/Services/SummarizerPreference.swift
@@ -38,7 +38,8 @@ final class SummarizerPreference {
     init(defaults: UserDefaults = .standard, key: String = defaultKey) {
         self.defaults = defaults
         self.keyName = key
-        self.type = defaults.string(forKey: key)
+        self.type =
+            defaults.string(forKey: key)
             .flatMap(SummarizerType.init(rawValue:)) ?? .onDevice
     }
 }

--- a/MindEcho/MindEcho/ViewModels/HomeViewModel.swift
+++ b/MindEcho/MindEcho/ViewModels/HomeViewModel.swift
@@ -49,7 +49,8 @@ class HomeViewModel {
             openAIAPIKey: openAIAPIKey)
     }
     @ObservationIgnored
-    var summarize: (String, String, SummarizerType, String) async throws -> String = { text, instruction, type, apiKey in
+    var summarize: (String, String, SummarizerType, String) async throws -> String = {
+        text, instruction, type, apiKey in
         try await SummarizationService.summarize(text: text, instruction: instruction, type: type, apiKey: apiKey)
     }
     @ObservationIgnored

--- a/MindEcho/MindEcho/ViewModels/TranscriptionViewModel.swift
+++ b/MindEcho/MindEcho/ViewModels/TranscriptionViewModel.swift
@@ -44,7 +44,8 @@ final class TranscriptionViewModel {
         SFSpeechRecognizer.requestAuthorization($0)
     }
     @ObservationIgnored
-    var summarize: (String, String, SummarizerType, String) async throws -> String = { text, instruction, type, apiKey in
+    var summarize: (String, String, SummarizerType, String) async throws -> String = {
+        text, instruction, type, apiKey in
         try await SummarizationService.summarize(text: text, instruction: instruction, type: type, apiKey: apiKey)
     }
     @ObservationIgnored

